orangewidget/report/report.py

```diff
@@ -230,24 +230,50 @@ def data(role=Qt.DisplayRole,
                             max_height=view.verticalHeader().defaultSectionSize()
                         )
                     elif isinstance(data_, QIcon):
-                        data_ = get_icon_html(data_, size=decoration_size)
+                        data_ = get_icon_html(data_, size=decoration_size) + " "
                     return data_
 
+                value = data()
+                if view is not None and col is not None:
+                    delegate = view.itemDelegateForColumn(col)
+                    if delegate is None:
+                        delegate = view.itemDelegate()
+                    if hasattr(delegate, "displayText"):
+                            value = delegate.displayText(value, QLocale())
+                value = value or ""
```

Comment:
There's an extra indent there that bothers me a little bit :)
Also, it confuses me that the delegate would be needed here. Shouldn't the model already be returning some nicely formatted text for DisplayRole?

Digging a bit deeper I see that specifically in the case of BarRatioTableModel, it changes DisplayRole to EditRole for some reason.

Reply:
> There's an extra indent there that bothers me a little bit :)

What lint doing?

(Hm, I see: no lint.)

I also thought that model would suffice, but at least some view use `displayText` for this.